### ISSUE TITLE
[Bug] Setting paintType1/paintType2 with vehicleProperties does not work with custom colors

### ISSUE THREAD

**Issue by hamcoremusic** [overextended/ox_lib#598]
**Describe the bug**
Currently I am trying to set a custom color matte on a vehicle, however no matter what I put into the paintType1/2 setting it doesn't change anything.

**To Reproduce**
1. Use the `ox_lib:setVehicleProperties` with this table
```lus
        model = 1934384720,
        dirtLevel = 0,
        plateIndex = 1,
        modSmokeEnabled = true,
        wheels = 1,
        modFrame = 0,
        modSpoilers = 2,
        modFender = 2,
        modRoof = 0,
        wheelColor = 0,
        paintType1 = 3,
        paintType2 = 3,
        tyreSmokeColor = {0, 175, 218, },
        modFrontBumper = 2,
        color1 = {0, 175, 218, },
        color2 = {0, 175, 218, },
```
2. Use something to set it while in car
3. The paint RGB changes, but it is not matte color type

**Expected behavior**
After setting the paint index to `3` I would expect that the custom color RGB would be changed to a matte paint type. It just sets it to `0` as far as I am seeing.

**Comment by hamcoremusic** [comment]
Additionally, if you set the car to a matte color first via color1 or any color menu, it will then custom color the matte

**Comment by jag3dagster** [comment]
I agree there is a weird conflict going on here. I will be digging into it further.
